backend/app/store/service/service.go

```diff
@@ -493,7 +493,8 @@ func (s *DataStore) HasReplies(comment store.Comment) bool {
 	for _, c := range comments {
 		if c.ParentID != "" && !c.Deleted {
 			if c.ParentID == comment.ID {
-				s.repliesCache.Set(comment.ID, true, cache.DefaultExpiration)
+				// renew the cache
```

Comment:
this is not exactly what it did before. I think you set nil to cause cache invalidation and make the next hit to get it? If so, this changes the logic and not the best way to do it. Probably calling Invalidate and get return true will be much closer.

_Generally speaking, this use case indicates the lack of somewhat useful Set method in lcw_

Reply:
It's storing value in the cache, same as it did before. My change is to store `nil` instead of `true` as we don't care about the value here anyway.

Reply:
well, adding a comment about it will be nice, because this logic is really not that obvious. Another thing - I fail to see how this code provides renew

Reply:
Make sense, I rewrote the comment: please look into it once more.